docs/api/leveraging-links-to-wikidata.md

```diff
@@ -0,0 +1,23 @@
+#### Link to Wikipedia and Wikidata {#link_to_wikipedia_and_wikidata}
```

Comment:
Your title must start with one of the diataxis section: How to, Explaining, Intro to, Tutorial, etc.

I would say this one is an explanation